test/parallel/test-global-descriptors.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+require('../common');
+
+const assert = require('assert');
+
+const {
+  value,
+  configurable,
+  enumerable,
+  writable
+} = Object.getOwnPropertyDescriptor(global, 'global');
+
+assert.strictEqual(value, global, 'global should be global object');
```

Comment:
This assertion isn't actually verifying that `global` is the global object. (If there was a bug and `global.global` was set to some other value, the assertion would still pass because `global` is being compared to itself.)

I think it would be better to compare `global` to something that is guaranteed to be the global object. For example:

```js
assert.strictEqual(value, (0, eval)('this'), 'global should be global object');
```